litmus-portal/frontend/src/views/Home/AgentConfiguredHome/AgentInfoContainer/styles.ts

```diff
@@ -0,0 +1,49 @@
+import { makeStyles } from '@material-ui/core';
+
+const useStyles = makeStyles((theme) => ({
+  agentInfoContainer: {
+    marginTop: theme.spacing(3.25),
+    padding: theme.spacing(3.125, 6.5),
+    display: 'flex',
+    alignItems: 'center',
+  },
+  agentInfoBlock: {
+    flexGrow: 1,
+  },
+  agentInfoData: {
+    display: 'flex',
+    padding: theme.spacing(1),
+    maxWidth: '25.5625rem',
+    height: '100%',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+
+    '& div': {
+      display: 'flex',
+      alignItems: 'baseline',
+
+      '& #agentCount': {
```

Comment:
let's use classnames instead of ids
just to have consistency in the UI development

Reply:
Sure, how would you recommend to handle such a situation then :thinking: should I give separate classnames?

Reply:
Point Being I didn't want to clutter the HTML for something that was reachable